source/user-manual/capabilities/vulnerability-detection/offline_update.rst

```diff
@@ -51,18 +51,24 @@ Alternatively, the feeds can be loaded from a local path. To achieve it, the ``p
 .. code-block:: xml
 
     <provider name="canonical">
-        <enabled>no</enabled>
-        <os url="http://local_repo/com.ubuntu.focal.cve.oval.xml">focal</os>
-        <os url="http://local_repo/com.ubuntu.bionic.cve.oval.xml">bionic</os>
-        <os url="http://local_repo/com.ubuntu.xenial.cve.oval.xml">xenial</os>
-        <os url="http://local_repo/com.ubuntu.trusty.cve.oval.xml">trusty</os>
+        <enabled>yes</enabled>
+        <os path="/local_path/com.ubuntu.focal.cve.oval.xml">focal</os>
+        <os path="/local_path/com.ubuntu.bionic.cve.oval.xml">bionic</os>
+        <os path="/local_path/com.ubuntu.xenial.cve.oval.xml">xenial</os>
+        <os path="/local_path/com.ubuntu.trusty.cve.oval.xml">trusty</os>
         <update_interval>1h</update_interval>
     </provider>
 
 Debian
 ^^^^^^
+The current implementation of Debian requires two different feeds at runtime in order to achieve the highest precision possible.
```

Comment:
Change this introduction and the note, same for the one for RedHat:

Currently, the Debian vulnerabilities are fetched from two different sources. One OVAL file for each supported Debian version, and a JSON feed with global information of the affected packages for each distribution. 

Both sources are necessary for the proper functioning of the scanner, how to configure each of them for an offline update is described below.